resources.py

```diff
@@ -125,7 +125,32 @@ def refresh_tokens():
             return jsonify({'error':'empty token'}), 400
         is_token_valid = jwt.decode(token_from_client, config['REFRESH_TOKEN_SECRET'], algorithms=["HS256"])
         user_id = is_token_valid['userid']
-        token = create_token(user_id)
+        token = state_machine.create_token(user_id)
         return jsonify(token), 200
     except:
-        return jsonify({'error':'invalid token'}), 400
\ No newline at end of file
+        return jsonify({'error':'invalid token'}), 400
+
+@resources.route('/documents', methods=['POST'])
```

Comment:
move all APIs under /documents to a new module (new dir named documents), create a blueprint for /documents, and register that blueprint in the main server file